### PR TITLE
Filter out CDATA tag for RSS feeds

### PR DESCRIPTION
**Pull request by bbashy**
## Description
Some RSS feeds use <![CDATA[]]> to house the link (guessing to escape some characters if needed).
This change checks if it's removed the CDATA tag and then returns the content inside.

## Related Issue
#688 

## How Has This Been Tested?
regexr.com only but willing to test an actual RSS feed to make sure.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jfurrow** [comment]
Honestly I have no idea if these feeds follow any convention. Does the `CDATA` property always contain a torrent URL?

**Comment by bbashy** [comment]
@jfurrow It's the only feed I see with CDATA being used but ruTorrent excludes CDATA stuff around links already.

Here's an example;
```
<link><![CDATA[https://sitedomain.org/rss/download/123456/file.name.torrent]]></link>
```

File: server/services/feedService.js [hunk 1]
```diff
@@ -163,6 +164,13 @@ class FeedService extends BaseService {
 
     // If there are no enclosures, then use the link tag instead
     if (feedItem.link) {
+      // remove CDATA tags around links
+      let cdata = regEx.cdata.exec(feedItem.link);
```
Comment (jfurrow) [inline server/services/feedService.js]: Use `const` instead of `let`

**Comment by bbashy** [comment]
@jfurrow Was that the only change required for styling?

**Pull request merged**